barista-quarkus-http/src/main/docker/Dockerfile.jvm

```diff
@@ -0,0 +1,54 @@
+####
+# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
+#
+# Before building the container image run:
+#
+# ./mvnw package
+#
+# Then, build the image with:
+#
+# docker build -f src/main/docker/Dockerfile.jvm -t quarkus/code-with-quarkus-jvm .
```

Comment:
These are generated code comments, but if someone reads them, they're going to use the wrong name..

Reply:
I've updated the comments